notes_app/client/app/notes/notes_service.js

```diff
@@ -39,11 +49,25 @@
           return false;
         })
       },
-      'pagenumToOffset': function(pagenum, limit) {
-        return (pagenum - 1) * limit;
+      'delete': function (id) {
+        return $http.delete(api_path + id + '/').then(function (response) {
+          for (var i = 0; i < data.notes.length; i++) {
+            if (data.notes[i].id === id) {
+              data.notes.splice(i,1);
+            }
+          }
+        }).catch(function errorHandler(response) {
+          $log.error('Problem with deleting the note');
+          return null;
+        });
+      },
+      'update': function () {
```

Comment:
the future might not come, it's redundant for this very moment. Please remove it